sdk/include/ndk/pstypes.h

```diff
@@ -991,6 +991,26 @@ typedef struct _EPROCESS_QUOTA_BLOCK
     ULONG ProcessCount;
 } EPROCESS_QUOTA_BLOCK, *PEPROCESS_QUOTA_BLOCK;
 
+//
+// Quota Structures
+//
+typedef enum _PS_QUOTA_TYPE
+{
+    PsNonPagedPool = 0,
+    PsPagedPool,
+    PsPageFile,
+#if (NTDDI_VERSION >= NTDDI_LONGHORN)
+    PsWorkingSet,
+    PsQuotaTypes,
+#elif (NTDDI_VERSION == NTDDI_LONGHORN)
+    PsWorkingSet,
+    PsCpuRate,
+    PsQuotaTypes,
+#else
+    PsQuotaTypes
+#endif
```

Comment:
Too many `PsQuotaTypes` :)
```suggestion
#if (NTDDI_VERSION >= NTDDI_LONGHORN)
    PsWorkingSet,
#elif (NTDDI_VERSION == NTDDI_LONGHORN)
    PsWorkingSet,
    PsCpuRate,
#endif
    PsQuotaTypes
```